vulnerabilities/tests/test_data/apache_kafka/test-advisories.json

```diff
@@ -0,0 +1,200 @@
+[
+    {
+        "aliases": [
+            "CVE-2022-34917"
+        ],
+        "summary": "This CVE identified a flaw where it allows the malicious unauthenticated clients to allocate large amounts of memory on brokers. This can lead to brokers hitting OutOfMemoryException and causing denial of service.",
+        "affected_packages": [
+            {
+                "package": {
+                    "type": "maven",
+                    "namespace": null,
+                    "name": "apache_kafka",
+                    "version": null,
+                    "qualifiers": null,
+                    "subpath": null
+                },
+                "affected_version_range": "vers:maven/>=2.8.0|<=2.8.1|!=2.8.2|>=3.0.0|<=3.0.1|!=3.0.2|>=3.1.0|<=3.1.1|!=3.1.2|>=3.2.0|<=3.2.1|!=3.2.3",
+                "fixed_version": null
+            }
+        ],
+        "references": [
+            {
+                "reference_id": "",
```

Comment:
Please put the CVE as a reference ID there as in CVE-2022-34917
The URL can be also https://kafka.apache.org/cve-list#CVE-2022-34917 as the page has ids in its HTML